src/Dodo.HttpClient.ResiliencePolicies/TimeoutPolicy/TimeoutPolicySettings.cs

```diff
@@ -2,16 +2,16 @@
 
 namespace Dodo.HttpClientResiliencePolicies.TimeoutPolicy
 {
-	public sealed class OverallTimeoutPolicySettings : ITimeoutPolicySettings
+	public class TimeoutPolicySettings
```

Comment:
Also, the second thought, maybe this is an excess class because we don't have it in any public constructor at ResiliencePolicySettings. How I as a client can use it?
Two way here:
1. Delete it (I know this was my idea to satisfy my wants for uniformity but it crashes by reality)
2. Make it internal 😕

Reply:
Yes, you are right about `sealed`. I will fix that.
I thought about this class too. Class itself looks nice because we keep all settings in the similar manner, but when I am working on tests refactoring, creating new instance each time and passing there single argument looks boring. I think we could keep it internal and use inside the library, but the end-user should provide only `TimeSpan` structures. I will fix that.

Reply:
I have made changes to this thing. I am playing a bit with TimeoutPolicy and decide to remove it. Code looks cleaner now. Also I leave only default constructor in the `ResiliencePoliciesSettings` because the overriden constructor solves the only problem of matching TimeSpan on TimeoutPolicySettings.